scripts/ci/gcp.sh

```diff
@@ -1,15 +1,12 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
```

Comment:
Could we maybe keep `set -eo pipefail`?

Reply:
@vikin91 it is still there, just a few lines down. Why this fixes the rvm problem 🤷